lib/rules/prop-types.js

```diff
@@ -301,6 +301,17 @@ module.exports = {
     }
 
     /**
+     * Removes quotes from around an identifier.
+     * @param {string} the identifier to strip
+     */
+    function stripQuotes(string) {
+      if (string[0] === '\'' || string[0] === '"') {
```

Comment:
Backticked strings require computed property brackets:
```js
{ [`complete?`]: number }
```
Does this not work with flow?